src/routes.ts

```diff
@@ -87,6 +88,22 @@ export default [
       ]
     }
   },
+  {
+    path: '/notice-of-withdrawal/:filingToBeWithdrawn',
```

Comment:
Good idea 👍

Background: the other local filings don't do this because their filing id (0 or xxx) was initially passed internally on the route, but now, since an external dashboard routes to local filings, the filing is a URL parameter.

What you've done here is better, but we're not going to change the others quite yet since the external dashboard would also need to be updated.